src/messaging/templates/messaging_dashboard.html

```diff
@@ -119,83 +118,91 @@ <h5 class="modal-title" id="newConversationModalLabel">Start New Conversation</h
         </div>
 
         <script>
+            const chatContainer = document.getElementById('chatContainer');
```

Comment:
Can we please add the javascripts in a separate file in static/js folder

Reply:
Moved Javascript to a separate file.